addons/purchase_stock/tests/test_anglo_saxon_valuation_reconciliation.py

```diff
@@ -548,6 +548,10 @@ def test_currency_exchange_journal_items(self):
         purchase_order.invoice_ids.action_post()
         post_bill_remaining_value = purchase_order.picking_ids.move_ids.stock_valuation_layer_ids.remaining_value
         self.assertEqual(post_bill_remaining_value, pre_bill_remaining_value)
+        currency_exchange_items = self.env['account.move.line'].search([
+            ('name', '=', 'Currency exchange rate difference'),
+        ])
+        self.assertFalse(currency_exchange_items)
```

Comment:
This is a weird test, we are checking for something that shouldn't exist in the first place. Maybe we can check the sum of all lines' credit to be what we expect it to be, so if there is some additional line it would show up.